platform-core-module/src/main/java/com/softicar/platform/core/module/CoreModule.java

```diff
@@ -47,4 +49,20 @@ public static EmfPagePath getParentPagePath() {
 
 		return new EmfPagePath().append(PARENT_FOLDER_TITLE);
 	}
+
+	//FIXME Should be removed when PLAT-1099 is done
```

Comment:
this is rather a TODO, there is no real problem with having there methods
```suggestion
	// TODO should be removed with PLAT-1099
```